app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -29,6 +29,12 @@
             android:layout="@layout/checkbox_left_preference"
             android:title="@string/preferences_sync_bookmarks" />
 
+        <CheckBoxPreference
```

Comment:
Still looking for the visibility to be false by default for these to prevent the flash and disappear.

Reply:
oh I misread your comment. I thought you meant default Check value